docs/technical-docs/actions.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Python CI Workflow
+parent: Technical Documentation
+---
+
+# Python CI Workflow
+
+## Overview
+
```

Comment:
I think we want 1-3 sentences, in the overview, to specify when/why people need to use GitHub actions. Right now, you have a great doc about how to use it. But I think we want something brief at the top at least to say when/why people HAVE TO use it.

It may or may not be worthwhle to have a section near the end too, to go over more detail of when/why to use it. I'll let you think that over and use your best judgment once you make the changes listed here. I do think we at least need something brief at the top make it clear when/why users will use this thing they are learning about. This is good to (re)communicate standards to the group, but also, to allow the reader to "visualize" themself using this, for their specific projects, when reading the doc. There is probably some educational psych lit that backs up why that is conducive to learning... but It just seems like a good idea if nothing else.